zscan_rr.go

```diff
@@ -136,7 +136,7 @@ func endingToTxtSlice(c chan lex, errstr, f string) ([]string, *ParseError, stri
 	return s, nil, l.comment
 }
 
-func setA(h RR_Header, c chan lex, o, f string) (RR, *ParseError, string) {
+func setA(h RR_Header, c chan lex, o, f string) (_ RR, e *ParseError, _ string) {
```

Comment:
That's lazy.. I should just reverse that and add var e *ParseError few lines lower. 
Named params just not going to be used, rr has to have type A and comment will also stay unused.